src/redux/reducers/authReducer.js

```diff
@@ -1,18 +1,4 @@
-import { combineReducers } from 'redux';
-import {
-  SIGNUP_REQUEST,
-  SIGNUP_FAILURE,
-  SIGNUP_SUCCESS,
-  LOGIN_REQUEST,
-  LOGIN_FAILURE,
-  LOGIN_SUCCESS,
-  LOGOUT_REQUEST,
-  LOGOUT_FAILURE,
-  LOGOUT_SUCCESS,
-  GET_USER_INFO_REQUEST,
-  GET_USER_INFO_FAILURE,
-  GET_USER_INFO_SUCCESS
-} from '../actions/authActions';
+import * as AuthActions from '../actions/authActions';
```

Comment:
I wish GitHub had the option to give little reactions to each line of code. This would get a thumbs up lol.....

Reply:
🤙!